### PR TITLE
Allow clang format binary to be specified with a path

### PR DESCRIPTION
**Pull request by fjricci**

### CODE REVIEW

File: Support/Testing/Travis/script.sh [hunk 1]
```diff
@@ -12,7 +12,13 @@
...
 
 cmake_package="cmake-3.4.0-Linux-x86_64"
-cformat="clang-format-3.8"
+
+if [[ -n "${CLANG_FORMAT_PATH-}" ]]; then
+  cformat="$CLANG_FORMAT_PATH"
+else
+  cformat="clang-format-3.8"
+fi
+
```
Comment (sas) [inline Support/Testing/Travis/script.sh]: `cformat="${CLANG_FORMAT_PATH-clang-format-3.8}"`

**Comment by sas** [comment]
@facebook-github-bot shipit

File: Support/Testing/Travis/script.sh (hunk 1 above)
Comment (fjricci) [inline Support/Testing/Travis/script.sh]: :open_mouth:

**Comment by strager** [comment]
Who would set `CLANG_FORMAT_PATH`?

**Comment by fjricci** [comment]
CentOS doesn't have access to llvm 3.8 packages. So I (we) use the clang-format built from our llvm trees

**Comment by fjricci** [comment]
Committed as f52152ef8b4bebc6c1aa94259611453946098148

**Pull request closed without merging**